### ISSUE TITLE
Support for gradle configuration cache

### ISSUE THREAD

**Issue by Gosunet** [Instabug/Instabug-Android#445]
Hello, 

We are using instabug in our android app. It's seems that instabug android is not compatible with [gradle configuration cache](https://docs.gradle.org/current/userguide/configuration_cache.html).
Gradle configuration cache is still an unstable feature but as of today instabug android is the only lib that we use that prevent us to use it.

Do you plan to support it ?

### Steps to Reproduce the Problem

Init a new project with instabug android lib.

Add this in gradle.properties : 

`org.gradle.unsafe.configuration-cache=true`

Build the app.

### Expected Behavior

No error building the app with gradle configuration cache

### Actual Behavior

Configuration cache problems found in this build.

```
1 problem was found storing the configuration cache.
- Plugin 'instabug-apm': registration of listener on 'Gradle.addListener' is unsupported
  See https://docs.gradle.org/7.4/userguide/configuration_cache.html#config_cache:requirements:build_listeners
```

### SDK Version

instabug = "11.7.0"

### Android Version

 compileSdkVersion = 33
 minSdkVersion = 21
 targetSdkVersion = 33

**Comment by IslamMohamady** [comment]
Hi @Gosunet 
Thanks for reporting. We are aware of the issue and are working on a fix for it. 
We will notify you once it is released 🙏🏻

**Comment by IslamMohamady** [comment]
Hi @Gosunet 
Thanks for reporting this. We will be looking into it and we will be sharing the updates with you ASAP

**Comment by maxharter** [comment]
Hi Instabug team,

Do you have any news about this issue please?
Thanks,

Maxime

**Comment by IslamMohamady** [comment]
Hi @maxharter 
This should be fixed in latest release 11.10.0 
I will close the ticket for now. And feel free to reopen it if you have any inquiry
